core/src/main/java/org/elasticsearch/common/util/BigArrays.java

```diff
@@ -392,9 +392,21 @@ public BigArrays(PageCacheRecycler recycler, @Nullable final CircuitBreakerServi
     /**
      * Adjust the circuit breaker with the given delta, if the delta is
      * negative, or checkBreaker is false, the breaker will be adjusted
-     * without tripping
+     * without tripping.
+     */
+    void adjustBreaker(final long delta) {
```

Comment:
How would you feel about removing this and making explicit the `dataAlreadyCreated` parameter?